### DISCUSSION TITLE
Choosing a max Delaunay distance

### DISCUSSION THREAD

**Discussion by rbutleriii** [giotto-suite/Giotto#860] [Q&A] [answered]
I noticed that the tutorials select particular distances for the `maximum_distance_delaunay` setting (e.g. `50` for the CosMx vs `400` for the Seqfishes), but the current function has an `auto` setting. Does that basically choose the best setting? In the papers it showed the graphs for `plotStatDelaunayNetwork` but I didn't find a description of how to make that deterrmination.

**Comment by jiajic** [comment, marked as answer]
Passing `auto` for this param means that the max network distance allowed will be automatically set as the largest delaunay distance that is not deemed an outlier. This maximal value is determined using the 5th value of `grDevices::boxplot.statsboxplot.stats()`, and should usually produce a reasonable max distance.

This documentation is present in the more specific `createSpatialDelaunayNetwork()`, but we should add it to `createSpatialNetwork()`
